### PR TITLE
v0.4.1 release aftermath

### PR DESCRIPTION
**Pull request by real-or-random**

### CODE REVIEW

**Comment by real-or-random** [comment]
cc @hebasto

File: tools/check-abi.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@
 set -eu
 
 default_base_version="$(git describe --match "v*.*.*" --abbrev=0)"
-default_new_version="master"
+default_new_version="HEAD"
```
Comment (hebasto) [inline tools/check-abi.sh:6]:
```suggestion
default_new_version="$(git rev-parse HEAD)"
```

**Review by hebasto — COMMENTED** [review]
Is it a proper moment to unify tag related steps in the "Regular release" and "Maintenance release" sections?

Regular release:
3. After the PR is merged, tag the commit and push it:

Maintenance release:
4. After the PRs are merged, update the release branch and tag the commit: 
5. Push tag:

**Comment by jonasnick** [comment]
ACK mod hebasto's comments

File: tools/check-abi.sh (hunk 1 above)
Comment (real-or-random) [reply tools/check-abi.sh:6]: I'm not entirely sure what you meant here, but I believe I fixed it

**Comment by real-or-random** [comment]
> Is it a proper moment to unify tag related steps in the "Regular release" and "Maintenance release" sections?

Done. Ready for review again.

**Comment by hebasto** [comment]
ACK b37fdb28ce373e359e4d3a9727693b724db31676.

**Review by jonasnick — APPROVED** [review]
ACK b37fdb28ce373e359e4d3a9727693b724db31676

**Pull request merged**